zap/src/output/tooling.rs

```diff
@@ -545,7 +545,7 @@ impl<'src> ToolingOutput<'src> {
 			.config
 			.evdecls
 			.iter()
-			.filter(|ev_decl| ev_decl.evty == EvType::Unreliable)
+			.filter(|ev_decl| matches!(ev_decl.evty, EvType::Unreliable(_)))
```

Comment:
The tooling file will need to deal with ordered event ID (see how we deal with `call_id`)